### PR TITLE
Fix: change percentage button for btc quote markets

### PR DESCRIPTION
**Pull request by oyershov**

### CODE REVIEW

File: src/components/PercentageButton/index.tsx [hunk 1]
```diff
@@ -19,6 +23,8 @@ export interface PercentageButtonProps {
 
 export const PercentageButton: React.FunctionComponent<PercentageButtonProps> = React.memo((props: PercentageButtonProps) => {
   return (
-      <button className={props.className || 'cr-percentage-button'} onClick={() => props.onClick(props.value)}>{`${props.value * 100}%`}</button>
+    <button className={props.className || 'cr-percentage-button'} onClick={() => props.onClick(props.calcValue || props.value)}>
```
Comment (akhlopiachyi) [inline src/components/PercentageButton/index.tsx:26]: fix indentation

File: src/components/OrderForm/index.tsx [hunk 2]
```diff
@@ -292,6 +294,14 @@ export class OrderForm extends React.PureComponent<OrderFormProps, OrderFormStat
         );
     }
 
+    private getPercentageButtonCalcValue = (value: number, to: string, type: FormType) => {
+        if (isFinexEnabled() && to.toLowerCase() === 'btc' && type === 'buy' && value === 1) {
```
Comment (calj) [inline src/components/OrderForm/index.tsx:298]: Don't hardcode anything!
Put just the percentage as props of the component.

Comment (oyershov) [reply src/components/OrderForm/index.tsx]: I can put exception settings (markets and coefficients) to constants file and pass as a props to order form. Should it be done this way?

**Comment by oyershov** [comment]
outdated

**Pull request closed without merging**